foundry.toml

```diff
@@ -1,7 +1,7 @@
 [profile.default]
 test = 'test/solidity'
-solc_version = '0.8.17'
-evm_version = 'shanghai'
+solc_version = '0.8.28'
```

Comment:
We need to wait for this PR otherwise types will fail https://github.com/lifinance/contracts/pull/981

Reply:
this was an error, I removed it.